extensions/machine-learning-services/src/packageManagement/packageManager.ts

```diff
@@ -195,7 +196,7 @@ export class PackageManager {
 	}
 
 	private async installPipPackage(requirementFilePath: string): Promise<string> {
-		let cmd = `"${this.pythonExecutable}" -m pip install -r "${requirementFilePath}"`;
+		let cmd = `"${this.pythonExecutable}" -m pip install --force-reinstall -r "${requirementFilePath}"`;
```

Comment:
We've removed force-reinstall from the other python install methods, in case this isn't required here.

Reply:
what was the reason? here I want to make sure newer version is installed if the older version is already there

---
In reply to: [400502190](https://github.com/microsoft/azuredatastudio/pull/9790#discussion_r400502190) [](ancestors = 400502190)

Reply:
If the version is newer, then it would get installed anyway. We wanted to avoid messing with a user's setup by forcing a reinstall, plus it would make the install take longer if the package was already present.

Reply:
ok I'll remove it. thanks

---
In reply to: [400507079](https://github.com/microsoft/azuredatastudio/pull/9790#discussion_r400507079) [](ancestors = 400507079)